data/Services.json

```diff
@@ -426,9 +462,9 @@
         ],
         "oracle": [
             {
-                "name": "Dedicated Compute Classic",
-                "ref": "https://cloud.oracle.com/compute-classic/dedicated/features",
-                "icon": "compute.png",
+                "name": "",
```

Comment:
name: Dedicated Virtual Machine Hosts
ref: https://docs.oracle.com/en-us/iaas/Content/Compute/Concepts/dedicatedvmhosts.htm

Reply:
Agreed, that would be the current offering.